cli/valet.php

```diff
@@ -248,6 +248,17 @@
 })->descriptions('Determine if this is the latest version of Valet');
 
 /**
+ * Switch between versions of PHP
+ */
+$app->command('switch-to [php_version]', function ($php_version) {
```

Comment:
Also, we're using camelCase variable names in this project (from what I checked, or we're being inconsistent 😄), so let's switch `$php_version` to `$phpVersion`.